producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -70,143 +76,137 @@ public class SdtEndpointPortTypeTest extends AbstractSdtUnitTestBase {
-                .andThrow(new RuntimeException("test"));
-        EasyMock.replay(mockBulkRequestHandler);
+    void testBulkFeedbackException() {
+        when(mockBulkRequestHandler.getBulkFeedback(any(BulkFeedbackRequestType.class)))
+                .thenThrow(new RuntimeException("test"));
 
         try {
             portType.getBulkFeedback(createBulkFeedbackRequestType());
-            fail("Runtime exception should have been thrown");
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
         } catch (final RuntimeException re) {
-            assertEquals("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");
         }
 
-        EasyMock.verify(mockBulkRequestHandler);
+        verify(mockBulkRequestHandler).getBulkFeedback(any(BulkFeedbackRequestType.class));
     }
 
     /**
      * Test submit query method completes successfully.
      */
     @Test
-    public void testSubmitQuerySuccess() {
-        EasyMock.expect(mockSubmitQueryHandler.submitQuery(EasyMock.anyObject(SubmitQueryRequestType.class)))
-                .andReturn(createSubmitQueryResponse());
-        EasyMock.replay(mockSubmitQueryHandler);
+    void testSubmitQuerySuccess() {
+        when(mockSubmitQueryHandler.submitQuery(any(SubmitQueryRequestType.class)))
+                .thenReturn(createSubmitQueryResponse());
 
         final SubmitQueryResponseType response = portType.submitQuery(createsubmitQueryRequestType());
 
-        assertNotNull("Response expected", response);
-        EasyMock.verify(mockSubmitQueryHandler);
+        assertNotNull(response, RESPONSE_EXPECTED);
+        verify(mockSubmitQueryHandler).submitQuery(any(SubmitQueryRequestType.class));
     }
 
     /**
      * Test submit query method handles exceptions successfully.
      */
     @Test
-    public void testSubmitQueryException() {
-        EasyMock.expect(mockSubmitQueryHandler.submitQuery(EasyMock.anyObject(SubmitQueryRequestType.class)))
-                .andThrow(new RuntimeException("test"));
-        EasyMock.replay(mockSubmitQueryHandler);
+    void testSubmitQueryException() {
+        when(mockSubmitQueryHandler.submitQuery(any(SubmitQueryRequestType.class)))
+                .thenThrow(new RuntimeException("test"));
 
         try {
             portType.submitQuery(createsubmitQueryRequestType());
-            fail("Runtime exception should have been thrown");
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
         } catch (final RuntimeException re) {
-            assertEquals("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");

```

Comment:
Might as well use version of assertEquals() that doesn't have the message parameter.